practices/house-style.md

```diff
@@ -21,6 +21,12 @@ Read this excellent [article by Nicholas C. Zakas](https://www.smashingmagazine.
 
 Read the [markup guide](../markup/house-style.md), the [JavaScript guide](../javascript/house-style.md) and the [CSS guide](../css/house-style.md) to understand our house style in these technologies. You should also familiarise yourself with our [Git](../git/git.md) strategies. The team you're deployed to may also have its own strategies in addition to these.
 
+### Indentation
+
+With regards to accessibility, we favour using `tab`s for indentation, rather than `space`s. Using `tab`s, every person can define their preferred _render width_ as they like. This also means, that we discourage setting any `tab width` settings in any shared configuration file, as that's a display thing, not a code thing.
```

Comment:
```suggestion
With regards to accessibility, we favour using `tab`s for indentation, rather than `space`s. Using `tab`s, every person can define their preferred _render width_ as they like. This also means that we discourage `tab width` settings in any shared configuration files, as that's a display thing, not a code thing.
```